### DISCUSSION TITLE
Is this the best place to post questions about Lighthouse reports and how to improve it?

### DISCUSSION THREAD

**Discussion by Elindo586** [GoogleChrome/lighthouse#15934] [Q&A]
I am for example running at:

50 Performance
96 Accessibility
78 Best Practices 
100 SEO
and a good check on PWA..

Of course I have a bunch of specific questions on how to get better performance and Best Practices scores.

**Comment by adamraine** [comment]
If you're only asking "How do I improve my scores?" then this isn't the best place. The Lighthouse report provides more information under each score to what's affecting it and how to improve it.
